app/uk/gov/hmrc/helptosave/connectors/EligibilityCheckConnector.scala

```diff
@@ -17,41 +17,41 @@
 package uk.gov.hmrc.helptosave.connectors
 
 import cats.data.EitherT
-import cats.instances.future._
-import cats.syntax.either._
-import com.google.inject.{ImplementedBy, Inject, Singleton}
-import play.api.Configuration
+import com.google.inject.{ImplementedBy, Singleton}
+import play.api.libs.json.{Format, Json}
 import uk.gov.hmrc.helptosave.config.WSHttp
-import uk.gov.hmrc.helptosave.models._
+import uk.gov.hmrc.helptosave.util.HttpResponseOps._
 import uk.gov.hmrc.helptosave.util.Result
 import uk.gov.hmrc.play.config.ServicesConfig
 import uk.gov.hmrc.play.http.HeaderCarrier
-import uk.gov.hmrc.helptosave.util.HttpResponseOps._
 
 import scala.concurrent.{ExecutionContext, Future}
 
-@ImplementedBy(classOf[ApiTwentyFiveCConnectorImpl])
-trait ApiTwentyFiveCConnector {
-  def getAwards(nino: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Result[List[Award]]
+private[connectors] case class EligibilityResult(isEligible: Boolean) extends AnyVal
+
+object EligibilityResult {
```

Comment:
if the case class is `private` might as well make the companion object `private` as well